test/e2e/utils/datainserter.go

```diff
@@ -74,11 +75,9 @@ func NewMultiDCDataInserter(dcHosts map[string][]string, options ...DataInserter
 		option(di)
 	}
 
-	if di.session == nil {
-		err := di.SetClientEndpoints(slices.Flatten(helpers.GetMapValues(dcHosts)))
-		if err != nil {
-			return nil, fmt.Errorf("can't set client endpoints: %w", err)
-		}
+	err := di.SetClientEndpoints(slices.Flatten(helpers.GetMapValues(dcHosts)))
```

Comment:
nit: can we move map helpers to a separate package (similar to pkg/helpers/slices) while we're at it? So `maps.Values` instead of `helpers.GetMapValues`, or use https://pkg.go.dev/golang.org/x/exp/maps#Values

Reply:
maybe not as part of this PR. It uses what existing codebase has.

Reply:
ok, doesn't have to be here